docs/source/model_design/general_reference_material/bliss/race_ethnicity/index.rst

```diff
@@ -79,10 +79,111 @@ on our `feedback form <https://docs.google.com/forms/d/e/1FAIpQLSeCED9TFQsH-1u4Q
 2.1 Data sources
 ----------------
 
-.. todo:: 
-
-    In this section we'll discuss the question "What do I do when data sources have different categories?" and how a health 
-    metrics researcher would answer this question and use that answer to inform their methods.
+As noted above, the government provides standardized race and ethnicity 
+categories, though these do change over time. Currently, the race 
+categories are American Indian or Alaska Native, Asian, Black or African American, 
+Native Hawaiian or Other Pacific Islander, and White. The ethnicity 
+categories are Hispanic or Latino and Not Hispanic or Latino. 
+
+By having a set standard, 
+it improves the chance that multiple data sources will have the same 
+categories. However, this is not guarunteed. Here, we will go through 
+a few scenarios of how race and ethnicity categories might be misaligned 
+between datasets and the options for handling each. 
+
+**Nested Categories**
```

Comment:
Should these be headers, so they are linkable/in the table of contents?

Reply:
I actually went back and forth on this myself. Are these important enough to be in the table of contents? But if you think it would be helpful happy to change.